stock-tracker/src/features/keyStats/component/keyStats.tsx

```diff
@@ -5,15 +5,19 @@ import ErrorMessage from "../../error/error";
 import "./KeyStats.css";
 import { AppState } from "../../../store/rootReducer";
 
+const numberWithCommas = (x: number) => {
+  return x.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ",");
+};
+
+const twoDecimalPlaces = (y: number) => {
+  return Math.abs(y).toFixed(2);
```

Comment:
nit: `x` isn't a great name for a variable...

Reply:
y?